src/plugins/visualize/public/application/utils/get_top_nav_config.tsx

```diff
@@ -318,24 +325,59 @@ export const getTopNavConfig = (
                   onTitleDuplicate,
                   returnToOrigin,
                 };
+
+                if (dashboardId) {
+                  // TODO: this should be an edit by value path
+                  // const appPath = `${VisualizeConstants.EDIT_PATH}/${encodeURIComponent(id)}`;
+                  const appPath = `${VisualizeConstants.LANDING_PAGE_PATH}`;
```

Comment:
Need to update this still

Reply:
I'm not sure if this will work the way you hope without major changes to Visualize by Value. Currently it's quite reliant on the by value attributes coming from the state transfer service, which pressing the back button won't restore.

I've been planning a follow-up to fix this issue:

1. Each editor adds a `{editorName}_lastByValue` key to the session storage
2. When leaving that editor in by value mode (implemented in `onAppLeave`) the current by value state is dumped into that key
3. On load of the `edit_by_value` url 
    - first check the state transfer for attributes
    - If the state transfer is empty, check the `{editorName}_lastByValue` Session storage key
    - if the Session storage key is also empty, redirect to 'create by value'

In the meantime, redirecting to the landing page is the best workaround.